rts/Rendering/GlobalRendering.cpp

```diff
@@ -250,6 +258,9 @@ CGlobalRendering::CGlobalRendering()
 	, sdlWindows{nullptr, nullptr}
 	, glContexts{nullptr, nullptr}
 {
+	screenViewMatrix = new CMatrix44f{};
+	screenProjMatrix = new CMatrix44f{};
```

Comment:
we really should use unique_ptr :P

Reply:
(it's not something I expect you to address in this PR; the codebase just doesn't use it for some reason)